src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -580,4 +607,13 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			this._viewModel[property] = {};
 		}
 	}
+
+	private createEditActionEvent(edit: DesignerEdit): ITelemetryEvent {
```

Comment:
Considering you only use this in one place I don't think you need to bother making an entire function for this